README.md

```diff
@@ -4,10 +4,11 @@ Retrieves and uses data from UBC Course Schedule.
 ## Developing
 Make sure you have `Go`, `make`, `Docker` installed
 
-Get the code (to your `$GOPATH`):
+To get the code (to your `$GOPATH`):
 ```
 go get -d github.com/smart-cs/scheduler-backend
 ```
+Your repository should end up in ~/go/src/github.com/smart-cs/scheduler-backend
```

Comment:
The default `$GOPATH` is `~/go`, what do you think about something like:

The default `$GOPATH` is `~/go` so the repository should end up in `~/go/src/github.com/smart-cs/scheduler-backend`, assuming your `$GOPATH` is `~/go`.

Reply:
How about:
By default, your $GOPATH is ~/go. If you haven't changed it, the repository should end up in ~/go/src/github.com/smart-cs/scheduler-backend

Reply:
Sound good to me, you can wrap $GOPATH, and the directories with back ticks -- ` to show it's code.

Reply:
`test`